docs/user_manual/calculations.md

```diff
@@ -231,10 +238,13 @@ For each iteration the following steps are executed:
 - Using LU decomposition, solve $J(i) \Delta x(i)  =  \Delta y(i)$ for $\Delta x(i)$
 - Compute $x(i+1)$ from $\Delta x(i) =  x(i+1) - x(i)$
 
-#### Iterative Current
+#### Iterative current power flow
+
+Algorithm call: {py:class}`CalculationMethod.iterative_current <power_grid_model.enum.CalculationMethod.iterative_current>`
 
 This algorithm is a Jacobi-like method for powerflow analysis.
-It has linear convergence as opposed to quadratic convergence in the Newton-Raphson method. This means that the number of iterations will be greater. Newton-Raphson will also be more robust in achieving convergence in case of greater meshed configurations. However, the iterative current algorithm will be faster most of the time.
+It has linear convergence as opposed to quadratic convergence in the [Newton-Raphson](#newton-raphson-power-flow) method. This means that the number of iterations will be greater.
+[Newton-Raphson](#newton-raphson-power-flow) will also be more robust in achieving convergence in case of greater meshed configurations. However, the iterative current algorithm will be faster most of the time.
```

Comment:
```suggestion
Additionally, [Newton-Raphson](#newton-raphson-power-flow) will be more robust converging in case of greater meshed configurations. Nevertheless, the iterative current algorithm will be faster most of the time.
```

Reply:
This is a problematic sentence: `meshed configuration` is a term but why `greater`? Unless this is a domain terminology, I suggest we come up with clearer wording; the `iterative current` faster in what sense? In terms of the runtime performance per iteration or in achieving similar convergence?